packages/plugin-commands-init/src/init.ts

```diff
@@ -65,17 +65,17 @@ export async function handler (
     'version',
     'description',
     'main',
+    'packageManager',
     'scripts',
     'keywords',
     'author',
     'license',
-    'packageManager',
```

Comment:
the location I used is the same as the one used by Yarn.

Reply:
There seems to be no clear standard for this. I think we can refer to our repository code.
https://github.com/pnpm/pnpm/blob/02054983227e41d07d2a2ee6f316f97aca929f41/package.json#L4

Reply:
I don't know. Maybe it makes sense if the field will be near the "dependencies" field as it is also describes a dependency.

Reply:
Okay, it's fine to keep it like this.